### ISSUE TITLE
Find Properties: Download button needs an expand/collapse state

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#655]
## Describe the task

Using a screen reader, when the "Download" button is activated, it doesn't announce anything to tell users what has happened. Similar to Filter button, the Download button acts like a disclosure. It should have an expanded/collapsed state

## Acceptance Criteria

-  use `<aria-expanded>` on Download button

**Comment by amyyeung17** [comment]
I can work on this :)

**Comment by CodeWritingCow** [comment]
@amyyeung17 here you go 😄
